about.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link    rel="stylesheet"    href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"    />    <link    rel="stylesheet"    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"    /><!-- font awesome icons -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;400&family=Poppins:wght@100&family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-element">
+            <div class="nav-items">
+                <ul class="social-icons">
+                    <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="">Logout</a></li>
+                </ul>
+                <div class="nav-bar">
+                    <div class="logo">
+                        <a href="index.html"><img src="./images/mlogo.png" alt="company logo"></a>
+                    </div>
+                    <ul class="menu">
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                    <button name="button" type="submit" class="btn">CC Campaign</button>
+                </div>
+            </div>   
+        </nav>
+    </header>
+
+    <section class="info-about">
+        <ul class="hamburger">
+            <li><a href="#" class="hamburger" id="bars">&#9776;</a></li>
+        </ul>
+        <div class="section-about">
+            <h4>"Hello! Music Lovers"</h4>
+            <h1>
+                THE MUSICAL EVENT OF THE YEAR
+                GLOBALMUZIK EASTER FIESTER 
+                2022
+            </h1>
+            <P class="about-para">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+                totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+                quibusdam magni aperiam consequatur!
+            </P>
```

Comment:
- [ ] Kindly increase the width of this section to match the given design on desktop screen.